app/controllers/viewhelper/EditLiability.scala

```diff
@@ -27,9 +27,9 @@ object EditLiability {
   }
 
   def returnTypeFromAmount(amount: BigDecimal): String = amount match {
```

Comment:
I think this can be refactored into an if statement if the value of the match is not required.

Reply:
It's fairly condensed - I'm not so sure it's a large issue

Reply:
Its not but I don't think it is good code style. It is effectively an if-thenIf-Else statement wrapped up in an unnecessary pattern match.

Reply:
I do personally feel it's fine